src/data/content/design-system/componenti/navscroll.yaml

```diff
@@ -81,7 +85,18 @@ tabs:
             title: Come usarlo
             headingLevel: 2
             text: |
-              _In stesura_
+              - Definisci le sezioni della pagina, utilizzando la intestazioni semantiche come `h2` oppure `h3`.
+              - Aggiungi nel componente le voci e i link necessari per navigare a ciascuna sezione della tua pagina.
+              - Assicurati che ogni link abbia un anchor tag che corrisponde all'ID della sezione di destinazione sulla pagina.
+              - Posiziona il componente a destra o a sinistra del contenuto principale della pagina, a seconda delle esigenze.
+              - Puoi aggiungere un effetto di scroll per rendere la navigazione più piacevole, utilizzando la [proprietà disponibile](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior) nei principali browser, senza fare ricorso a Javascript.
+
+              ### Attenzione a
+              - Testare il comportamento di scroll su vari dispositivi per assicurarti che sia fluido e funzioni come previsto.
```

Comment:
```suggestion
              - Testare il comportamento di scroll su vari dispositivi per assicurarti che sia fluido e funzioni come previsto.
              - Testare con utenti la navigazione da tastiera o con strumenti assistivi come lettori di schermo, poni particolare attenzione al focus al termine dell'azione di scorrimento e a come accedere o tornare al componente.  
```

Reply:
non riesco a cambiare io direttamente il testo, posso fare solo commit. il secondo punto, possiamo dividerlo in 2 separati? oppure, se è un unico punto, dopo "lettori di schermo" andrebbe messo un punto fermo (.). la frase è molto lunga e densa.